AndelaOSP/art-dashboard
# Added the initial frontend documents

## What does this PR do?
Adds the initial documentation to guide frontend workflow

## Description of Task to be completed?
Added frontend documents that explains the best practices we use

## How should this be manually tested?
Read through the documents and verify that there are no broken links, and the explanations make sense. 

## What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162048024

## Any background context you want to add?
The tightly-coupled logic in the frontend is what prompted the need for guidelines to use when coding.

## Important notes

## Packages installed

## Deployment note

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
